libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -203,13 +205,14 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _sem->give();
     }
 
+    // log raw data if flag is set
```

Comment:
That makes sense, but not being a "professional" C programmer I'll defer to the original authors on such changes and just delete the comment.